### PR TITLE
Add intergration test for freeze and thaw

### LINKED ISSUE #495: Add integration tests: freeze and thaw unit
# Integration tests

_Goal:_ Verify that a unit can be frozen and thawed. 

__Notes:__
- Use the `bluechictl` CLI to freeze and thaw
- Use `systemctl` output to check the frozen/thawed status

Requires #489  to be merged (or done alongside it)

### PR DESCRIPTION
**Pull request by irishair**
Add a test case for verifying that a unit can be frozen and thawed.

Fixes: #495

### CODE REVIEW

**Review by engelmi — APPROVED** [review]
LGTM

File: tests/bluechi_test/container.py [hunk 1]
```diff
@@ -89,6 +89,18 @@ def service_is_active(self, unit_name: str) -> bool:
         result, _ = self.exec_run(f"systemctl is-active {unit_name}")
         return result == 0
 
+    def service_is_frozen(self, unit_name: str) -> bool:
```
Comment (ygalblum) [inline tests/bluechi_test/container.py]: These two methods can be combined with an additional parameter `state`. Something like this:
```python
def is_service_freezer_state(self, unit_name: str, state: str) -> bool:
  _, output = self.exec_run(f"systemctl show {unit_name} | grep FreezerState")
  return state in str(output)
```

Comment (pypingou) [reply tests/bluechi_test/container.py]: if you go this way you could even parametrize "FreezerState" no?

Comment (irishair) [reply tests/bluechi_test/container.py]: Two methods combined to `def get_unit_freezer_state(self, unit_name: str) -> str`.

Comment (engelmi) [reply tests/bluechi_test/container.py]: Looks good, thanks! @irishair 
Merging now :)

**Pull request merged**